Problem3.js

```diff
@@ -0,0 +1,22 @@
+// Time Complexity : O(n)
+// Space Complexity : O(n)
```

Comment:
Kindly optimize space complexity. Do not try to use extra space in Linked list problems.